help/en/package/apps/util/issuereporter/swing/IssueReporter.shtml

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
+"http://www.w3.org/TR/html4/strict.dtd">
+
+<html lang="en">
+<head>
+  <meta name="generator" content=
+  "HTML Tidy for Mac OS X (vers 31 October 2006 - Apple Inc. build 15.17), see www.w3.org">
+
+  <title>JMRI: Issue Reporter</title>
+  <meta name="author" content="Bob Jacobsen">
+  <meta name="keywords" content="JMRI help Issue Reporter">
+  <!-- Style -->
+  <meta http-equiv="Content-Type" content="text/html; charset=us-ascii">
+  <link rel="stylesheet" type="text/css" href="/css/default.css"
+  media="screen">
+  <link rel="stylesheet" type="text/css" href="/css/print.css"
+  media="print">
+  <link rel="icon" href="/images/jmri.ico" type="image/png">
+  <link rel="home" title="Home" href="/">
+  <!-- /Style -->
+</head>
+
+<body>
+  <!--#include virtual="/Header.shtml" -->
+  <div class="nomenu" id="mBody">
+    <div id="mainContent">
+
+      <h1>JMRI: Issue Reporter</h1>
+      <p>The JMRI Issue Reporter allows you
+        to create an entry in JMRI's 
+        <a href="https://github.com/JMRI/JMRI/issues">Issue tracking system</a>
+        for either a bug report or a feature request.</p>
+        
+    <p>In order to create an entry, you need a (free)
+        GitHub account. If you don't have one yet, follow
+        <a href="https://github.com/join?ref_cta=Sign+up">this link</a>
+        and sign up for one.</p>
+        
+    <a href="Report_Issue__Window.png"><img src="Report_Issue__Window.png" align="right" width="264" height="256" ></a> 
+    <p>In your default web browser, sign on to the 
+        <a href="https://github.com/login">GitHub web site</a>.</p>
+    
+    <p>From the JMRI Help menu, select "Report Issue...".
+        A window like the one on the right will open (click to make the images larger).
+        <ul>
+            <li>Select whether you're reporting a bug or making feature request
+            <li>Leave JMRI selected as the report destination
+            <li>Add a useful title.  That really helps getting attention to your item.
+            <li>Add a detailed description.  (You can add screen shots to it later on)
+            <li>Click the check boxes for the information you want to send 
+                along with the report. We recommend you check all three.
+            <li>Then click the "Submit to GitHub" button
+        </ul>
+        
+    <p>A GitHub window will open in your browser. 
```

Comment:
Wouldn't it be good if all the `<p>` tags have an ending `</p>` tag?
I apologize if I'm nitpicking. It's not my intention.

Reply:
A closing `</p>` is not mandated and is more of an XHTML -ism.

Reply:
Thanks for the explanation of both tags.